src/arch/xtensa/include/arch/lib/cache.h

```diff
@@ -75,6 +78,8 @@ static inline void dcache_writeback_invalidate_all(void)
 
 #endif /* __ARCH_LIB_CACHE_H__ */
 
```

Comment:
should this comment __ARCH_LIB_CACHE_H be moved to the endif below? Also maybe it worth to also add a comment about !_ASSEMBLER && !LINKER after endif.